x-pack/plugins/actions/server/mocks.ts

```diff
@@ -32,10 +31,20 @@ const createStartMock = () => {
       .fn()
       .mockReturnValue(actionsAuthorizationMock.create()),
     preconfiguredActions: [],
+    renderActionParameterTemplates: jest.fn(),
   };
   return mock;
 };
 
+// this is a default renderer that escapes nothing
```

Comment:
Not a real mock :-).  But had to do this, as there was so much other code dependent on the templating, which would have required a LOT more non-trivial changes to other unrelated tests.  Function tests have been added to test a complete end-to-end of alerts firing actions and capturing the results of the action executions.